docs/reference/content/bson/extended-json.md

```diff
@@ -117,31 +117,31 @@ The `Document.parse()` static factory method constructs an instance of a `JsonRe
 equivalent Document. `JsonReader` automatically detects the JSON flavor in the string, so you do not need to specify it.
 
 ### Reading and Writing JSON Directly
-If you do not need a document and only want to deal with JSON, you can use `JsonString` to read and write JSON directly. `JsonString` 
-is simply a wrapper class that takes in a `String` in the constructor and returns the `String` in the get `getJson()` method. 
-Reading and writing directly is more efficient than constructing a `Document` first and then calling `toJson()`, and it is also more efficient than calling `Document#parse`. 
-The codec responsible for reading/writing JSON (`JsonStringCodec`) is part of the default registry, so doing this is very simple 
+If you do not need a document and only want to deal with JSON, you can use `JsonObject` to read and write JSON directly. `JsonObject` 
+is simply a wrapper class that takes in a `String` in the constructor and returns the trimmed `String` in the get `getJson()` method. 
```

Comment:
Nit: Change `get getJson()` to `getJson()` here and below.

Reply:
Done